crates/bevy_internal/src/lib.rs

```diff
@@ -141,3 +141,17 @@ pub mod dynamic_plugin {
 
 #[cfg(target_os = "android")]
 pub use ndk_glue;
+
+pub mod prelude2d {
+    #[cfg(feature = "bevy_core_pipeline")]
+    pub use crate::core_pipeline::ClearColor;
+    pub use crate::prelude::*;
+    #[cfg(feature = "bevy_sprite")]
+    pub use crate::sprite::prelude::*;
+}
+
+pub mod prelude3d {
+    #[cfg(feature = "bevy_pbr")]
+    pub use crate::pbr::prelude::*;
+    pub use crate::prelude::*;
+}
```

Comment:
```suggestion
pub mod prelude2d {
    pub use crate::prelude::*;

    #[cfg(feature = "bevy_core_pipeline")]
    pub use crate::core_pipeline::ClearColor;
    #[cfg(feature = "bevy_sprite")]
    pub use crate::sprite::prelude::*;
}

pub mod prelude3d {
    pub use crate::prelude::*;

    #[cfg(feature = "bevy_pbr")]
    pub use crate::pbr::prelude::*;
}
```
This makes it easier to see what `#[cfg]` applies to every `use` at a glance.